cartographer_ros/cartographer_ros/trajectory_options.cc

```diff
@@ -50,6 +50,12 @@ TrajectoryOptions CreateTrajectoryOptions(
   options.provide_odom_frame =
       lua_parameter_dictionary->GetBool("provide_odom_frame");
   options.use_odometry = lua_parameter_dictionary->GetBool("use_odometry");
+  options.fixed_frame_pose_frame =
+      lua_parameter_dictionary->GetString("fixed_frame_pose_frame");
+  options.provide_fixed_frame_pose_frame =
```

Comment:
This option is not needed, please also remove it in lua configs and so on.
(It appears to be consistent with odometry on first glance, but provide_odom_frame is unrelated, it controls whether TrajectoryState::local_to_map will be published.)

Reply:
Done.